src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,140 @@
 package core.basesyntax;
 
+import edu.emory.mathcs.backport.java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    Object [] elements;
+    private int size = 0;
```

Comment:
initialize in the constructor, but what is the default value of int variable?

Reply:
done!